code/core/src/test/testing-library.ts

```diff
@@ -114,6 +114,9 @@ type _UserEvent = typeof _userEvent;
 
 export interface UserEvent extends _UserEvent {}
 
+// @ts-expect-error CJS workaround
+export const uninstrumentedUserEvent = _userEvent.default ?? _userEvent;
```

Comment:
Considering we'll embark on the ESM migration soon, maybe we can leave a few more breadcrumbs on what the problem is, and how this fixes it?

Reply:
I think it has something to do with our prebundling and our build setup. But I have no clue why this is needed.